doc/operating-principle.md

```diff
@@ -144,12 +148,12 @@ The accelerator unit set the appropriate signals on the C-Mem request channel an
 - `cmem.q_hart_id` the processor core's hart ID is used for routing the C-Mem request accross the accelerator interconnect.
   This signal is *not* forwarded to the offloading core accross the X-Mem interface.
 
+An accelerator subsystem can send only one memory instruction back to a core at any  time.
```

Comment:
What is the goal of this limitation? If no load/store access faults are possible, what would be the reason to limit memory bandwidth